### ISSUE TITLE
cannot spawn bots in OSX

### ISSUE THREAD

**Issue by falcondai** [Marqt/ViZDoom#81]
I could not get bots to spawn in `cig_bots.py` on OSX but it works fine on my Ubuntu machine. anything I missed? as a side question, how can i get more detailed logs from vizdoom?

**Comment by wjaskowski** [comment]
Indeed. Something is wrong on OSX.


--
Wojciech Jaśkowski, Ph.D.
Laboratory of Intelligent Decision Support Systems
Institute of Computing Science
Poznan University of Technology, Poland
http://www.cs.put.poznan.pl/wjaskowski

**Comment by falcondai** [comment]
I am impressed that most of it works. that really helps with the interactive aspects in my development experience. 

a separate question: i notice that `vizdoom` might not exit when i interrupt the python script that called `game.init()` on both Ubuntu and OSX. I have to `kill -9` to terminate them for now. Is there a way to properly close `vizdoom` in case of an interrupt? I don't mind cleaning up on my local machine but it worries other users on a shared cluster.

**Comment by wjaskowski** [comment]
Could you provide a minimal example which works that way? If I interrupt
the process by ctrl+c, I get ``RuntimeError: Signal SIGINT received.
ViZDoom instance has been closed.`` and no vizdoom process is left alone
(on OSX at least)

**Comment by wjaskowski** [comment]
As a inelegant workaround, to make bots work on OSX copy ``bots.cfg`` into ``ViZDoom/bin/vizdoom.app/Contents/MacOS/zcajun/``

**Comment by falcondai** [comment]
the SIGINT issue seems to be related to my use of tensorflow training (testing code seems to exit properly when interrupted). i will report back with a minimal example later. the workaround works for me.

**Comment by Marqt** [comment]
I experienced the same inconvenience with Caffe.
Unfortunately deep learning frameworks like to register their own signal handlers that overrides ViZDoom handler or vice versa. To solve this, handlers chaining should be implemented on both sides (this is in my TO DO list for ViZDoom).

The best solution for now is to implement your own signal handler, that will override other handlers, stop training and call `game.close()`.

---

Fixed - search path for bots.cfg are now the same as in Linux version (working dir and exe dir).
